dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/internal/HttpChain.java

```diff
@@ -456,6 +457,10 @@ public synchronized void update(String resolvedHostName) {
                             enableSameSite = true;
                             chanProps.put(HttpConfigConstants.PROPNAME_SAMESITE_STRICT, samesiteOptions.get("strict"));
                         }
+                        if (samesiteOptions.containsKey("partitioned")) {
```

Comment:
Do we need a test where we just set "partitioned=true" in the server.xml and then use the Servlet APIs to add samesite?

Reply:
Yes, that's a good test case. That scenario not work with the current code, but I'll test it out.  It will need to be added in the 6.1 FAT. I have a few tests planned interactions with Cookie#setAttribute behavior.

Reply:
Was a test added?

Reply:
Also why would Servlet 6.1 FAT be required?

Reply:
Typo -- I mean servlet 6.0 fat (which I add tests too).

Reply:
Yes -  6.0 FAT: testSetAttributeWithPartitionedConfig

https://github.com/OpenLiberty/open-liberty/pull/27809/files/0ab5adce5a8f6f4d0a83f18d02d8cec49ad3cdc5#diff-f5ca25689c46ddee6312a8127f6d541acd8db6ade26af722bd7f0f52313c513fR135